pkg/devicetemplates/ttscsv.go

```diff
@@ -393,6 +438,20 @@ func (*ttsCSV) Convert(ctx context.Context, r io.Reader, ch chan<- *ttnpb.EndDev
 			EndDevice: dev,
 			FieldMask: ttnpb.FieldMask(paths...),
 		}
+
+		if t.profile != nil && t.profile.ShouldFetchProfile(dev) {
+			profileTmpl, err := t.profile.FetchProfile(ctx, dev)
+			if err != nil {
+				return err
+			}
+			if err := profileTmpl.EndDevice.SetFields(tmpl.EndDevice, tmpl.FieldMask.Paths...); err != nil {
+				return err
+			}
+			tmpl.FieldMask.Paths = ttnpb.AddFields(tmpl.FieldMask.Paths, "version_ids.vendor_id", "version_ids.vendor_profile_id")
```

Comment:
Shouldn't you copy the `version_ids` also to `profileTmpl`? This line should be before the `SetFields` right?

Reply:
You are correct, kind of an embarrassing mistake